src/firefly/js/util/WebUtil.js

```diff
@@ -39,6 +39,10 @@ export function getProp(key, def) {
     return get(GLOBAL_PROPS, [key], def);
 }
 
+
+
+export const isDefined= (x) => x!==undefined;
```

Comment:
lodash has _isUndefined_ function, we use _!isUndefined_ all over.

Reply:
I know, I don't like `isUndefined` because we are always reversing the logic. I think it makes the code more messy.  I was reading some discussion with the creator of lodash, he does not even like `isUndefined` because it is so easy to implement.  That is why does not want to add `isDefined`. When I was reading that I realized the I would really prefer `isDefined`.